web/src/message_scroll.js

```diff
@@ -14,6 +14,17 @@ import * as unread_ui from "./unread_ui";
 let hide_scroll_to_bottom_timer;
 export function hide_scroll_to_bottom() {
     const $show_scroll_to_bottom_button = $("#scroll-to-bottom-button-container");
+    const $scroll_to_bottom_clickable_area = $("#scroll-to-bottom-button-clickable-area");
+
+    // delete the tippy when the hide transition is finished
+    $show_scroll_to_bottom_button[0].addEventListener("transitionend", () => {
+        const tooltip = $scroll_to_bottom_clickable_area[0]._tippy;
+        // make sure the tooltip exists and the class is not currently showing
+        if (tooltip && !$show_scroll_to_bottom_button.hasClass("show")) {
+            tooltip.destroy();
+        }
+    });
```

Comment:
This seems like the correct solution. Let's move this to `initialize` function below so that we don't create hundreds of event listeners here.

Also, let's verify the `propertyName` of the transition.

Reply:
Thank you for the review! I've made the requested changes.